pubsublite/config.go

```diff
@@ -211,6 +212,121 @@ func (tc *TopicConfigToUpdate) toUpdateRequest() *pb.UpdateTopicRequest {
 	}
 }
 
+// ExportDestinationConfig is the configuration for exporting to a destination.
+// Implemented by *PubSubConfig.
+type ExportDestinationConfig interface {
+	setExportConfig(ec *pb.ExportConfig) string
+}
+
+// PubSubConfig configures messages to be exported to a Pub/Sub topic.
+// Implements the ExportDestinationConfig interface.
+type PubSubConfig struct {
```

Comment:
Maybe align the naming for this to closer match the interface it is going to fullfill PubSubExportDestinationConfig?

Reply:
Renamed to `PubSubDestinationConfig` to compromise on length and clarity :)